com.unity.render-pipelines.core/Runtime/RenderGraph/RenderGraphBuilder.cs

```diff
@@ -61,11 +78,25 @@ public TextureHandle ReadTexture(in TextureHandle input)
         public TextureHandle WriteTexture(in TextureHandle input)
         {
             CheckResource(input.handle);
+            m_Resources.IncrementWriteCount(input.handle);
             // TODO RENDERGRAPH: Manage resource "version" for debugging purpose
             m_RenderPass.AddResourceWrite(input.handle);
             return input;
         }
 
+        /// <summary>
+        /// Specify a Texture resource to read and write to during the pass.
+        /// </summary>
+        /// <param name="input">The Texture resource to read and write to during the pass.</param>
+        /// <returns>An updated resource handle to the input resource.</returns>
+        public TextureHandle ReadWriteTexture(in TextureHandle input)
+        {
+            CheckResource(input.handle);
```

Comment:
Seems like this one is missing the IncrementWriteCount.
Wasn't it possible to put it directly in AddResourceWrite in the end?

Reply:
Ah yes, moved it later and forgot to update this. 

I need m_Resources to call the Increment; will update in a sec